### DISCUSSION TITLE
is Browser secure exam from screen record like Atlas browser ?

### DISCUSSION THREAD

**Discussion by muhamedRadwan** [SafeExamBrowser/seb-mac#311] [Q&A]
is Browser secure exam from screen record like Atlas browser ?
when students make screen record on Atlas it shows a gray screen but when I try safeExamBroswer if I opened screen share before starting the app it complete normal and share exam questions

**Comment by danschlet** [comment]
You have to either add this Atlas browser to your settings as a [prohibited process](https://safeexambrowser.org/macosx/mac_usermanual_en.html#ProhibitedProcessesSection). Or you can start using the [Assessment Mode AAC](https://safeexambrowser.org/macosx/mac_usermanual_en.html#SecurityPane) kiosk mode (supported by macOS since 10.15.4, possibly with some restrictions as described in [Release Notes](https://safeexambrowser.org/macosx/mac_release_notes_en.html) of SEB 3.1.1 (DNS pre-pinning in your SEB configuration might be necessary on some macOS versions before macOS 12.1).

I'll have a look at that browser which I didn't know until now and will add it as a default prohibited process to the next SEB version 3.3.

**Comment by danschlet** [comment]
Is Atlas browser an actual macOS app? I didn't find such a product. Do you have a link where this is available?
